src/purging_rmw.c

```diff
@@ -246,16 +246,30 @@ get_then_time(struct dirent *entry, const char *entry_path)
  */
 int
 purge (const st_waste * waste_curr)
+{
+  return empty_option_purge(waste_curr, 0);
+}
+
+/*!
+ * Purges files older than x number of days, unless purge_after is set to
+ * 0 in the config file.
+ * @param[in] waste_curr the linked list of waste folders
+ * @param[in] empty_trash forces the trash to be completely emptied
+ * @return error number
+ * @see is_time_to_purge
+ * @see get_then_time
+ */
+int
+empty_option_purge (const st_waste * waste_curr, bool cmd_empty)
 {
   short status = 0;
 
-  bool cmd_empty = 0;
   bool cmd_dry_run = 0;
   char *rmwtrash_env = getenv("RMWTRASH");
```

Comment:
This isn't needed at all because we won't be relying on the RMWTRASH env var anymore

Reply:
I have removed dry_run and `char *rmwtrash_env = getenv("RMWTRASH");` from purging_rmw.c .
RMWTRASH is still available in rmw.c for setting a fake year.

Reply:
@Jammyjamjamman I'm sorry I gave you bad info there. I didn't notice it was still being used for dry run. Could you please revert